suite-native/module-onboarding/src/navigation/OnboardingStackNavigator.tsx

```diff
@@ -16,6 +16,7 @@ export const OnboardingStackNavigator = () => (
     <OnboardingStack.Navigator screenOptions={stackNavigationOptionsConfig}>
         <OnboardingStack.Screen
             name={OnboardingStackRoutes.Onboarding}
+            // @ts-expect-error
```

Comment:
@bouchja1

Reply:
yes, fixed in 9b886964e9f5d5c074a74a6d7e11e217a23115a7